lib/bulbs-elements/util/prep-reading-list-analytics.js

```diff
@@ -0,0 +1,19 @@
+import prepGaWrapper from './prep-ga-wrapper';
+import getAnalyticsManager from './get-analytics-manager';
+
+export default function prepReadingListAnalytics (element, dimension12) {
+  let parent = element.closest('bulbs-reading-list-item')[0];
+
+  if (parent) {
+    this.parentAnalyticsDimensions = parent.dataset.contentAnalyticsDimensions;
+    this.href = parent.dataset.href;
+    this.title = parent.dataset.title;
+  }
+
+  let dimensionObject = {
+    dimension12: dimension12,
+  };
+
+  this.analyticsManager = getAnalyticsManager();
+  this.analyticsWrapper = prepGaWrapper(this.parentAnalyticsDimensions, dimensionObject);
```

Comment:
this is true, was experimenting with this pattern - since most of the logic in `prepReadingListAnalytics` is initializing/setting values on `this` it seemed worth extracting out. 

would it be more clear if the method had `setValues` in the name? or should this be something that should just be copied in each element?